nextflow_schema.json

```diff
@@ -31,13 +31,16 @@
             "properties": {
                 "lineage": {
                     "type": "string",
-                    "default": "auto"
+                    "default": "null",
```

Comment:
Is the expectation that this string is set to "null" or the non-string value null? Former is truth-y in boolean terms and the latter is false-y.

Reply:
I basically just need this to be a required parameter that the user gives that doesn't already have a default. Because of issues with auto taking up a bunch of resources I don't want to set that as default. So I guess I need it to be null without ""?

Reply:
Yeah if you want it to be required, without quotes is the way to go. Otherwise, the UI will allow the user to proceed with "null"